base/libgit2/blame.jl

```diff
@@ -0,0 +1,35 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+function GitBlame(repo::GitRepo, path::AbstractString; options::BlameOptions=BlameOptions())
+    blame_ptr_ptr = Ref{Ptr{Void}}(C_NULL)
+    @check ccall((:git_blame_file, :libgit2), Cint,
+                  (Ptr{Ptr{Void}}, Ptr{Void}, Cstring, Ptr{BlameOptions}),
+                   blame_ptr_ptr, repo.ptr, path, Ref(options))
+    return GitBlame(repo, blame_ptr_ptr[])
+end
+
+function Base.count(blame::GitBlame)
+    return ccall((:git_blame_get_hunk_count, :libgit2), Int32, (Ptr{Void},), blame.ptr)
```

Comment:
how many different measures can a blame object be counted by? this seems too context-less and ambiguous to extend Base.count